fog/view/server/src/shard_responses_processor.rs

```diff
@@ -72,25 +73,31 @@ mod tests {
     use mc_common::logger::{test_with_logger, Logger};
     use mc_util_grpc::ConnectionUriGrpcioChannel;
 
-    fn create_successful_mvq_response() -> MultiViewStoreQueryResponse {
+    fn create_successful_mvq_response(client_index: usize) -> MultiViewStoreQueryResponse {
         let mut successful_response = MultiViewStoreQueryResponse::new();
         let client_auth_request = Vec::new();
         successful_response
             .mut_query_response()
             .set_data(client_auth_request);
+        let view_uri_string = format!(
+            "insecure-fog-view-store://node{}.test.mobilecoin.com:3225",
```

Comment:
Nit: Should this be
```
format!("{}://node{}.test.mobilecoin.com:3225",
FogViewStoreScheme::SCHEME_INSECURE,
client_index)
```

Reply:
Done. Also did the same for the port number.